srcpkgs/tcl/template

```diff
@@ -1,13 +1,15 @@
 # Template file for 'tcl'
 pkgname=tcl
 version=8.6.10
-revision=2
+revision=3
 wrksrc="tcl${version}"
 build_wrksrc=unix
 build_style=gnu-configure
 configure_args="--enable-threads --without-tzdata --enable-man-symlinks
  --disable-static --disable-rpath --with-system-sqlite
- tcl_cv_strtod_unbroken=ok"
+ tcl_cv_strstr_unbroken=ok
+ tcl_cv_strtod_unbroken=ok
```

Comment:
I didn't find `strtod` in the configure output, maybe only `strstr` and `strtoul` are required?